src/AzureOpenAIProxy.Management/Components/Pages/ModelEdit.razor.cs

```diff
@@ -62,7 +73,7 @@ private async Task OnValidSubmit(ModelEditorModel model)
         m.Location = model.Location!;
         m.Active = model.Active;
 
-        await DbContext.SaveChangesAsync();
+        await ModelService.UpdateOwnerCatalogAsync(m.CatalogId, m);
```

Comment:
```suggestion
        await ModelService.UpdateOwnerCatalogAsync(m);
```

Let's not bother pass in the `CatalogId`, since the whole model is passed in.